### PR TITLE
Developer center integration (I) - Overview / Support / Examples (I)

### PR DESCRIPTION
**Pull request by VictorVelarde**
- Overview
- Support
- Examples (I)
  - [x]  basic
  - [x]  deckgl
  - [x]  basemap

+ upgrade to airship 2.4

### CODE REVIEW

**Comment by VictorVelarde** [comment]
@borja-munoz maybe you can check the PR (specially the browser support / versions... .md documents)

Regarding to the examples, the review will be probably easier in a deployment to dev. center from 'develop', which I would have hopefully soon.

**Comment by VictorVelarde** [comment]
I will deal with the other block of examples (style / interactivity / dataview) in a different PR

File: docs/support/03-browser-support.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
 ## Browser support
 
+Web SDK requires a browser that supports WebGL, a method of generating dynamic 3D graphics using JavaScript, accelerated through hardware (GPU).
```
Comment (borja-munoz) [inline docs/support/03-browser-support.md]: I would say "dynamic 2D and 3D graphics"

File: docs/support/03-browser-support.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
 ## Browser support
 
+Web SDK requires a browser that supports WebGL, a method of generating dynamic 3D graphics using JavaScript, accelerated through hardware (GPU). 
 
+Regarding to the rendering, Web SDK is based on **deck.gl** library, and that implies a good support on a broad base of browsers, including common mobile devices.
```
Comment (borja-munoz) [inline docs/support/03-browser-support.md]: I would change it to "Regarding rendering"

File: docs/support/04-versions.md [hunk 3]
```diff
@@ -1,2 +1,48 @@
...
+As a workaround, you can use an older version by using a fixed version. See the previous point.
+
+In case you were relying on undocumented or deprecated features, you should consider that workaround a temporal measurement. You should also update your application, so it no longer uses them.
+
+On the other hand, if you are facing a newly introduced bug, you are welcome to [open an issue](https://github.com/cartodb/web-sdk/issues/new) or use other [support options](/developers/web-sdk/support/support-options/).
+
+
+### Updates frequency
+
+Although we tend to do a release every several weeks of work, we don't have a fixed schedule for new Web SDK releases. However, if we planned to release a specific feature, we might delay a release. Also, in the case of critical regressions, we will make a bug fix release before the next major/minor release happens.
```
Comment (borja-munoz) [inline docs/support/04-versions.md]: I would change it to "release every few weeks"

**Review by borja-munoz — APPROVED** [review]
Just minor comments

**Pull request merged**